### DISCUSSION TITLE
Markdown Table Of Contents links don't work in the Android app

### DISCUSSION THREAD

**Discussion by jacko-yang** [community/community#101163] [Mobile] [answered]
### Select Topic Area

Bug

### Body

Some Markdown documents have a table of contents consisting of links of the following form.

```
- [Section 1](#section-1)
- [Section 2](#section-2)
```

When viewing the document as HTML, tapping such links is supposed to automatically scroll to the section that is linked. However, as of version 1.145.0 of the Android app, tapping _any_ TOC link just scrolls to the top of the document instead of the linked section.

This issue was previously brought up in [this discussion](https://github.com/orgs/community/discussions/21200#discussion-4242123), and it was subsequently fixed for the iOS app (TOC links behave correctly on my iPad). I'm not sure if the issue was just never fixed in the Android version of the app or if an unrelated regression is causing this issue, but the current behavior of TOC links seems to be broken on my Android phone.

Additional info:
Example Markdown document with a TOC: https://github.com/AlanWalk/markdown-toc#4-configuration
Software: GitHub for Android, version 1.145.0. Running on Android 14.
Device: Samsung Galaxy S22+

**Comment by eliperkins** [comment, marked as answer]
Thanks for this report, @jacko-yang! I can understand how frustrating it would be for links like these to not scroll to the correct content, especially on long READMEs. I've filed a bug report on our internal tracking repo for this issue. I'll work with the engineering team to make sure it's addressed in an upcoming update of the Android app. 🙇

**Reply by jacko-yang** [reply]
Seems like the issue has been fixed. Thanks!

**Reply by BA-CalderonMorales** [reply]
Still seeing this issue on S26 Ultra.

**Reply by TuTu-15728** [reply]
TOC links still scrolls to the top (Android 16, App version 1.252.0). Any updates?
